askama_derive/src/generator.rs

```diff
@@ -1200,7 +1200,7 @@ impl<'a> Generator<'a> {
         };
 
         buf.write(&format!(
-            "::askama::filters::markdown({}, ",
+            "::askama::filters::markdown({}, &",
```

Comment:
Normally, only this change should be needed and not the rest. Also a lot more types are actually implementing `AsRef<str>`, so this is a regression in a sense since now it only works with `String` and `str`.

Reply:
Yeah, that's exactly what I'm worried about too.